### PR TITLE
FilePicker supports download links

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- [x] Update schemas to support downloadLink in node input/output
- [x] FilePicker also renders a text box to support downloadLink. This is serialized/deserialized.

## Related issue number
related to ITISFoundation/osparc-issues#343

DL

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: api/specs/common/schemas/project-v0.0.1.json [hunk 1]
```diff
@@ -200,6 +200,21 @@
...
+                    },
+                    {
+                      "type": "object",
+                      "additionalProperties": false,
+                      "required": [
+                        "downloadLink"
+                      ],
+                      "properties": {
+                        "downloadLink": {
+                          "type": "string"
```
Comment (sanderegg) [inline api/specs/common/schemas/project-v0.0.1.json:212]:
```suggestion
                          "type": "string",
                          "format": "uri"
```

File: api/specs/common/schemas/project-v0.0.1.json [hunk 2]
```diff
@@ -273,6 +288,21 @@
...
```
Comment (sanderegg) [inline api/specs/common/schemas/project-v0.0.1.json:300]:
```suggestion
                          "type": "string",
                          "format": "uri"
```

**Review by sanderegg — APPROVED** [review]
great, will do the node ports asap

**Comment by odeimaiz** [comment]
> great, will do the node ports asap

👍

**Review by pcrespov — APPROVED** [review]

**Pull request merged**